lib/actions/status.js

```diff
@@ -2,28 +2,50 @@ const { find } = require("lodash");
 const migrationsDir = require("../env/migrationsDir");
 const config = require("../env/config");
 
-module.exports = async db => {
+module.exports = async (db, shouldLoadFromDatabase) => {
```

Comment:
This parameter will be true when calling this function to show the status list and when doing down migrations if the new config param is true. It will be false for up migrations as we don't care about saved content in that case.